salt/metalk8s/defaults.yaml

```diff
@@ -1,9 +1,11 @@
 ---
-cluster: kubernetes
-
-metalk8s: {}
+metalk8s:
+  endpoints:
+    salt-master:
+      ip: 127.0.0.1
```

Comment:
This is not a 'default'. The value is supposed to be in the pillar. If it's not, there's nothing you can 'fall back' on, it's a failure.

Rather fail with "I don't know the Salt master's address" than to configure a minion with this arbitrary IP address, and then the minion not properly functioning.

Reply:
It's kind of default value if you have no master endpoint, which is the case when you configure minion the first time, then we use localhost as default.

Btw if we want to specify this we will need to overwrite it in every salt call in the bootstrap command line because we only have it at the end of the bootstrap script

Reply:
Honestly, I prefer to add some more data in the 'injected' pillar in the bootstrap script rather than have arbitrary 'defaults'/fallbacks.